lang/lua-copas/patches/makefile.patch

```diff
@@ -1,15 +1,15 @@
---- lua-copas-1.2.1_org/Makefile	2014-06-04 16:39:17.451563827 +0800
-+++ lua-copas-1.2.1/Makefile	2014-06-04 16:39:41.115563309 +0800
-@@ -1,10 +1,10 @@
- # $Id: Makefile,v 1.3 2007/10/29 22:50:16 carregal Exp $
+--- a/Makefile
++++ b/Makefile
+@@ -3,10 +3,10 @@
+ DESTDIR ?= 
  
  # Default prefix
--PREFIX = /usr/local
-+PREFIX ?= $(DESTDIR)/usr
+-PREFIX  ?= /usr/local
++PREFIX  ?= $(DESTDIR)/usr
  
  # System's lua directory (where Lua libraries are installed)
--LUA_DIR= $(PREFIX)/share/lua/5.1
-+LUA_DIR= $(PREFIX)/lib/lua
+-LUA_DIR ?= $(PREFIX)/share/lua/5.1
++LUA_DIR ?= $(PREFIX)/lua/5.1
```

Comment:
Is this patch even needed?  that default won't work anyway, so it must be getting properly set by the openwrt toolchain already.

Reply:
Nope. good call.